### PR TITLE
Update README to indicate that we're dead.

### PR DESCRIPTION
**Pull request by syvb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**NOTE: This project is dead. See #559 for more information, including a V2.**
```
Comment (PlasmaPower) [inline README.md]: I'd prefer "this repository is dead". The project as a whole encompasses V2 IMO.

**Comment by PlasmaPower** [comment]
> Don't bother voting, the bot is down.

I'd prefer to vote anyway, we can still merge the PR and it's nice to get a consensus first.

Edit: the statement was in the OP, but has been removed.

**Comment by Redmega** [comment]
I agree with @PlasmaPower. We should at least require reviews & allow some time for voting before merging while the bot is sleeping.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+**NOTE: This repository is dead. See #559 for more information, including a V2.**
```
Comment (andrewda) [inline README.md]: I believe you have to manually link issues in the README unless it's been changed in the past couple weeks. Might be nice to add that link so it's easier to access the issue.

**Review by andrewda — DISMISSED** [review]

**Review by andrewda — APPROVED** [review]

**Comment by mark-i-m** [comment]
Can we make it say  "... dead _for now_ ..." instead? This makes it sound as if people shouldn't try to contribute.

**Comment by PlasmaPower** [comment]
@mark-i-m I think when we restart it will be with a new code base.

**Comment by mark-i-m** [comment]
Ah, got it

**Pull request merged**